barretenberg/ts/src/log.ts

```diff
@@ -0,0 +1,17 @@
+import { ILogObj, Logger } from 'tslog';
```

Comment:
Why not `pino`?

Reply:
`pino` requires being browserified or a few polyfills. I think we really want browser bb.js to be as polyfill free as possible, and it does very little logging too.

Reply:
Mmm, you might be right, it isn't pino, but our usage of it. Trying now

Reply:
Pino works 😄